benchmark_runner/benchmark_operator/benchmark_operator_workloads.py

```diff
@@ -41,6 +42,16 @@ def __init__(self, kubeadmin_password: str = '', es_host: str = '', es_port: str
         if environment_variables.environment_variables_dict['kubeadmin_password']:
             self.__oc.login()
 
+    @property
+    def runner_version(self):
+        """getter"""
+        return self._runner_version
+
+    @runner_version.setter
+    def runner_version(self, value):
```

Comment:
Give this a different name so that someone doesn't inadvertently call the wrong one (e. g. `set_runner_version`).